example/image_example.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import imageio
+from scipy import ndimage
+import os
+
+
+def rotate_and_blur(image_path):
+    full_path = os.path.join(os.path.dirname(__file__), image_path)
+    img_array = imageio.imread(full_path)
+
+    '''
+    this can be replaced with `np.flipud(img_array)` or a related
```

Comment:
I have never seen people use string literals in place of block comments like this, except for docstrings.  That said, I looked at PEP8 and it doesn't seem to address this:

https://www.python.org/dev/peps/pep-0008/#comments

I think these should probably be made into block comments.

Also, we should probably use full sentences.

Reply:
Sounds good to me.

Reply:
At first, I thought that using triple-quote string literals would be worse bc it would generate code, but it seems that this is not the case:

https://sgillies.net/2017/05/30/python-multi-line-comments-and-triple-quoted-strings.html

Thus, there is not really any technical reason to prefer one vs the other besides convention it seems.

Reply:
This case is a little bit like a docstring, in a way? I realized that for long comments one reason I use triple-quotes is because the syntax highlighting is brighter for them.